kb/2025-09-08/how_to_reduce_migration_time_of_vms.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'How to reduce migration time of VMs'
```

Comment:
I think the title is a bit too generic, making it sounds like some migration performance optimization KB. Seems to me the intention here is to force an immediate rescheduling of the VM when the host is down.

Maybe a title like "Force Migrate Unreachable Virtual Machines During Node Outage".